content-review/impl/urkund/src/main/java/org/sakaiproject/contentreview/urkund/UrkundSubmissionData.java

```diff
@@ -0,0 +1,35 @@
+package org.sakaiproject.contentreview.urkund;
+
+import java.util.Date;
+import java.util.Map;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class UrkundSubmissionData {
+
+	public Integer Id;
+	public Date Timestamp;
+	public String ExternalId;
+	public String Filename;
+	public String MimeType;
+	public Map<String, Object> Status;
+	public Map<String, Object> Document;
+	public Map<String, Object> Report;
+	public String Subject;
+	public Boolean Anonymous;
```

Comment:
Is there a link to a reported problem with Jackson deserialization and variable names? This doesn't mesh with my experience using jackson + json... Just curious.